packages/compass-connections/src/components/connection-list/connection-list.tsx

```diff
@@ -141,11 +147,31 @@ function ConnectionList({
   removeAllRecentsConnections: () => void;
   duplicateConnection: (connectionInfo: ConnectionInfo) => void;
   removeConnection: (connectionInfo: ConnectionInfo) => void;
+  openConnectionImportExportModal: (modal: FavoriteAction) => void;
 }): React.ReactElement {
-  const [recentHeaderHover, setRecentHover] = useState(false);
+  const [recentHoverProps, recentHeaderHover] = useHoverState();
+  const [favoriteHoverProps, favoriteHeaderHover] = useHoverState();
 
   const { theme } = useTheme();
 
+  const favoriteActions = useMemo(() => {
+    const actions: ItemAction<FavoriteAction>[] = [];
+
+    actions.push({
+      action: 'import-favorites',
+      label: 'Import saved connections',
+      icon: 'Download',
+    });
+
+    actions.push({
+      action: 'export-favorites',
+      label: 'Export saved connections',
+      icon: 'Export',
+    });
```

Comment:
Any reason we are pushing into the array here in a memo inside the render method? These seem to be constant, this can be defined outside of the component I think, unless I'm missing something

Reply:
Just the fact that I lazily copied this from `connections.tsx` (and I imagine there it was copied from other places where a prop _did_ influence the action list). :slightly_smiling_face: I took the liberty to apply your suggestion in both places, totally makes sense.